cloudimg/ms_azure.py

```diff
@@ -17,6 +17,10 @@
 
 from cloudimg.common import BaseService, PublishingMetadata
 
+# Disable noisy Azure HTTPS requests debug/info logs
+azure_logger = "azure.core.pipeline.policies.http_logging_policy"
+logging.getLogger(azure_logger).setLevel(logging.WARNING)
```

Comment:
I don't think this should be here.  I think configuring loggers at import-time is an antipattern (and generally having side-effects on import is an antipattern).

- you don't know in what order modules are being imported. If anyone else does the same, it's hard to know if your configuration takes effect.
- just because your module is being imported, does not imply it's going to be used by the current process.

Also, I don't think something like this belongs in the library at all.  A library like cloudimg can be used in different contexts and it's not for this library to assume that all users of it would always be uninterested in those logs.

What problem is this intended to solve?  Neither debug nor info logs will do anything by default anyway, so it seems like debug or info level are being *enabled* somewhere.  Maybe in one of your tasks using this library?  If so, I believe it would make more sense to adjust this log level there, i.e. at the same place you increase verbosity of everything by default, you avoid increasing verbosity of this undesired logger.

If it's intended to adjust the log levels when running in Pub production, I would rather stick with the centralized Pub mechanism for that (log levels can be adjusted in target settings), so that the services owners have the ability to change the log levels without code updates.

Reply:
+1 for such change in pubtools or in "centralized Pub mechanism"

But IMHO it make a sense to document/recommend  logger setting for `azure.core.pipeline.policies.http_logging_policy`
somewhere in this project.

Reply:
Sure!

I'll close this PR to do it in the `pubtools`.

The idea is to suppress some Request/Response JSON logs which should be on `DEBUG` but for whatever reason it's implemented as `INFO` in Azure SDK.

Thanks for your feedback!

Reply:
@JAVGan Would you mind to report issue to upstream as well ?
IIUC it should be 
* https://github.com/Azure/azure-sdk-for-python

Reply:
Sure!

Reply:
https://github.com/Azure/azure-sdk-for-python/issues/29215